inc/BrainPadDisplay.h

```diff
@@ -0,0 +1,27 @@
+#ifndef BPDISPLAY_H
+#define BPDISPLAY_H
+#include "Image.h"
+#include "MbedI2C.h"
+
+namespace codal {
+
+   class BrainPadDisplay {  
+  
+   uint8_t Data[2];
+   int DeviceAddress = 0x78;
+   typedef unsigned char byte;
+
+     public:
+     codal::_mbed::I2C i2c;
+     BrainPadDisplay(Pin& sda, Pin& scl);
+    
+    // Initialize the display
+    void InitScreen();
```

Comment:
Is this proper to be its own function in codal or should it be done in the class ctor?

Reply:
Not sure, we copied what the other codal repos did, if it was similar.